### PR TITLE
[New Rule] Suspicious Adobe Acrobat Updates Service Child Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/885

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ user.name:root and
+ not process.executable:
+                  (/Library/PrivilegedHelperTools/com.adobe.ARMDC.SMJobBlessHelper or
+                   /usr/bin/codesign or
+                   /private/var/folders/zz/*/T/download/ARMDCHammer or
+                   /usr/sbin/pkgutil or
+                   /usr/bin/shasum or
+                   /usr/bin/perl* or
+                   /usr/sbin/spctl or
+                   /usr/sbin/installer)
```
Comment (bm11100) [inline rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml:40]:
```suggestion
event.category:process and event.type:(start or process_started) and
  process.parent.name:com.adobe.ARMDC.SMJobBlessHelper and
  user.name:root and
  not process.executable:
  (
    /Library/PrivilegedHelperTools/com.adobe.ARMDC.SMJobBlessHelper or
    /usr/bin/codesign or
    /private/var/folders/zz/*/T/download/ARMDCHammer or
    /usr/sbin/pkgutil or
    /usr/bin/shasum or
    /usr/bin/perl* or
    /usr/sbin/spctl or
    /usr/sbin/installer
  )
```

Comment (bm11100) [reply rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml:40]: no query changes, just spacing

File: rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to exploit privilege escalation vulnerabilities related to Adobe Acrobat Reader PrivilegedHelperTool
+responsible for installing updates. For more information refer to the following CVE's - CVE-2020-9615, CVE-2020-9614 and
+CVE-2020-9613 and verify that the impacted system is patched.
```
Comment (bm11100) [inline rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml:11]:
```suggestion
Detects attempts to exploit privilege escalation vulnerabilities related to the Adobe Acrobat Reader PrivilegedHelperTool
responsible for installing updates. For more information, refer to CVE-2020-9615, CVE-2020-9614 and
CVE-2020-9613 and verify that the impacted system is patched.
```

File: rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+"""
+false_positives = ["Trusted System or Adobe Acrobat Related Processes."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Adobe Acrobat Updates Service Child Process"
```
Comment (bm11100) [inline rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml:18]:
```suggestion
name = "Suspicious Child Process of Adobe Acrobat Reader Update Service"
```

File: rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to exploit privilege escalation vulnerabilities related to the Adobe Acrobat Reader PrivilegedHelperTool
+responsible for installing updates. For more information, refer to CVE-2020-9615, CVE-2020-9614 and
+CVE-2020-9613 and verify that the impacted system is patched.
+"""
+false_positives = ["Trusted System or Adobe Acrobat Related Processes."]
```
Comment (bm11100) [inline rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml:13]:
```suggestion
false_positives = ["Trusted system or Adobe Acrobat Related processes."]
```

**Review by bm11100 — APPROVED** [review]

**Review by threat-punter — CHANGES_REQUESTED** [review]
Looks good but it looks like the linting was missed

**Comment by Samirbous** [comment]
> Looks good but it looks like the linting was missed

I usually relint once I receive last comments/approvals (due to potential many changes in-between reviews)

**Review by devonakerr — APPROVED** [review]

File: rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  (
+    /Library/PrivilegedHelperTools/com.adobe.ARMDC.SMJobBlessHelper or
+    /usr/bin/codesign or
+    /private/var/folders/zz/*/T/download/ARMDCHammer or
+    /usr/sbin/pkgutil or
+    /usr/bin/shasum or
+    /usr/bin/perl* or
+    /usr/sbin/spctl or
+    /usr/sbin/installer
+  )
```
Comment (brokensound77) [inline rules/macos/privilege_escalation_exploit_adobe_acrobat_updater.toml:42]: kql query whitespace is normalized at package generation time and so it is better to split them in a specific manner

```suggestion
event.category:process and event.type:(start or process_started) and
  process.parent.name:com.adobe.ARMDC.SMJobBlessHelper and
  user.name:root and
  not process.executable: (/Library/PrivilegedHelperTools/com.adobe.ARMDC.SMJobBlessHelper or
                           /usr/bin/codesign or
                           /private/var/folders/zz/*/T/download/ARMDCHammer or
                           /usr/sbin/pkgutil or
                           /usr/bin/shasum or
                           /usr/bin/perl* or
                           /usr/sbin/spctl or
                           /usr/sbin/installer)
```

**Review by brokensound77 — APPROVED** [review]
Query format change and 🚢

**Comment by Samirbous** [comment]
@threat-punter relinted :)

**Comment by threat-punter** [comment]
> > Looks good but it looks like the linting was missed
> 
> 
> 
> I usually relint once I receive last comments/approvals (due to potential many changes in-between reviews)

Roger that @Samirbous. Thanks 👍

**Pull request merged**